packages/react-mutation-mapper/src/util/FilterUtils.ts

```diff
@@ -152,6 +153,20 @@ export function applyDefaultProteinImpactTypeFilter(
     );
 }
 
+export function applyDefaultProteinChangeFilter(
```

Comment:
i see there is an establishe convention here but i think the naming isn't great.   the name ```applyDefaultProteinChangeFilter``` suggests something that actually applies it but this is just a test, so maybe ```shouldApplyDefaultProteinChangeFilter``` ?   If it's not too much work to change the names of these things, lets do it.

Reply:
Good point. Name choice is not great. This function (and any other applyFilter function) actually just tests the mutation for a given filter and returns a boolean value. We should probably fix the naming in another PR though.